.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    if: "${{ !contains(github.event.pull_request.title || github.event.issue.title, 'chore(main): release') }}"
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+      issues: write
+    steps:
+      - name: Release Please  
+        uses: google-github-actions/release-please-action@v4
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: simple
+
+      - uses: actions/checkout@v4
+
+      - name: Debug release outputs
```

Comment:
do we need this steps?

Reply:
hey yes, often times we need to dig into what was the output of release-please, as their docs usually miss details that we can only grasp at runtime